src/components/functional_module/src/plugin_manager.cc

```diff
@@ -193,6 +193,12 @@ bool PluginManager::IsHMIMessageForPlugin(application_manager::MessagePtr msg) {
   Json::Value value;
   Json::Reader reader;
   reader.parse(msg->json_message(), value);
+
+  if (value.isMember(("result"))) {
```

Comment:
@LitvinenkoIra according to [our specification](https://github.com/smartdevicelink/sdl_core/blob/master/src/components/interfaces/Json_HMI_message_specification.txt) "result" in HMI response is always object so I propose to move this check somewhere to upper level, for example in `ApplicationManagerImpl::Handle`

Reply:
@AKalinich-Luxoft Done aaaf361da4c8fe81873c35a9b61ab8f247ae89ec